modules/core/package.json

```diff
@@ -75,7 +75,7 @@
     "sanitize-html": "^1.27.5",
     "secp256k1": "^4.0.2",
     "secrets.js-grempe": "^1.1.0",
-    "stellar-sdk": "^8.1.0",
+    "stellar-sdk": "^8.2.5",
```

Comment:
@CapnMigraine @argjv

Reply:
is there a fix for the drone issue in the newer version, or is this purely speculative?

Reply:
this is me just guessing

Reply:
for posterity, the flaky CI failures should indeed be fixed by upgrading: https://github.com/stellar/js-stellar-sdk/issues/645